lib/KDropdownMenu.vue

```diff
@@ -145,9 +146,11 @@
       window.removeEventListener('keydown', this.handleOpenMenuNavigation, true);
     },
     methods: {
-      handleOpen() {
+      async handleOpen() {
```

Comment:
Was this a change made from the linter?  :0

Reply:
For some reason the linter didn't like the double nested `$nextTick` call unless it used `await this.$nextTick()` twice instead - so this seemed like the easiest way to workaround it. It feels like maybe a bug in the linter, but this also feels more readable, so I didn't see a concern.

Reply:
Cool!